### PR TITLE
docs(changelog) introduce changelog and add 0.2.0 changes

### PR DESCRIPTION
**Pull request by hbagdi**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Table of Contents
+
+- [0.2.0](#020---20180924)
+- [0.1.0](#010---20180615)
+
+##  [0.2.0] - 2018/09/24
+
+- :warning: Dropped metrics that were aggregated across services in  Kong.
```
Comment (p0pr0ck5) [inline CHANGELOG.md]: double whitespace before `Kong`

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  These metrics can be obtained much more efficiently using queries in Prometheus.
```
Comment (p0pr0ck5) [inline CHANGELOG.md]: Any docs/pointers on how that can be accomplished in prometheus?

Comment (hbagdi) [reply CHANGELOG.md]: No, we don't have docs for it. We expect users to be able to already know about it or figure that out from the documentation hosted by Prometheus.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+##  [0.2.0] - 2018/09/24
+
+- :warning: Dropped metrics that were aggregated across services in Kong.
+  These metrics can be obtained much more efficiently using queries in Prometheus.
+  [#8](https://github.com/Kong/kong-plugin-prometheus/pull/8)
+
+##  [0.1.0] - 2018/06/15
+
+- Initial release of Prometheus plugin for Kong
```
Comment (p0pr0ck5) [inline CHANGELOG.md]: For consistency can we add a period at the end?

**Review by p0pr0ck5 — APPROVED** [review]
LGTM apart from one thing to address at merge

**Review by fffonion — APPROVED** [review]
LGTM

**Comment by coopr** [comment]
Does this update suggest that https://docs.konghq.com/hub/kong-inc/prometheus/ also get updated? (Not sure if it does or doesn't, just raising the question, thanks)

**Comment by hbagdi** [comment]
Yes, I've a personal todo to update that documentation and also Grafana Dashboard.

**Pull request merged**